rose/client/car.py

```diff
@@ -1,10 +1,10 @@
-import random
+from random import randrange
```

Comment:
This change is not needed and unwanted. We like to import only modules, to make it easy to understand dependencies.